Qase_Test/Src/Pages/Project/CreateNewProjectPage.cs

```diff
@@ -0,0 +1,46 @@
+using OpenQA.Selenium;
+using Qase_Test.Core.Browser.Service;
+using Qase_Test.Pages.Base;
+
+namespace Qase_Test.Pages.Project
+{
+    public class CreateNewProjectPage : BasePage
+    {
+        private const string ErrorCodeMessage = "alert-message";
```

Comment:
perhaps this locator is applicable to other pages, if so, it is more logical to put it in base